Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixInfo.tsx

```diff
@@ -0,0 +1,134 @@
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import { Button, Modal, ModalBody, Table } from "reactstrap";
+
+interface IProps {
+    bigfixId: string;
+}
+
+interface IState {
+    bigfixModal: boolean;
+    computerInfo: object;
+    isFetched: boolean;
+}
+
+export default class EquipmentBigFixInfo extends React.Component<IProps, IState> {
+    public static contextTypes = {
+        fetch: PropTypes.func,
+        permissions: PropTypes.array,
+        router: PropTypes.object,
+        team: PropTypes.object
+    };
+
+    constructor(props) {
+        super(props)
+        this.state = { bigfixModal: false, computerInfo: {}, isFetched: false };
+    }
+
+    public render() {
+        return (
+            <>
+                <label> Bigfix Id</label>
+                <span />
+                {this._renderInfoIcon()}
+                {this._renderBigFixModal()}
+            </>
+        );
+    }
+
+    private _renderInfoIcon = () => {
+        if (this.props.bigfixId) {
```

Comment:
small thing/just an fyi, but i usually try to do:
```
if(!this.props.bigfixId){
  return null;
}
return (...)
```

Reply:
fixed